doc/installingLocal.md

```diff
@@ -11,13 +11,14 @@ This page describes the procedure to run PrairieLearn within Docker, but using a
 git clone https://github.com/PrairieLearn/PrairieLearn.git
 ```
 
-* Install the Node.js packages:
+* Install the Node.js packages.  This will use the version of `npm` that is pre-installed in the Docker image, so you don't need your own copy installed.
 
 ```sh
-cd PrairieLearn
-npm install
+docker run -w /PrairieLearn -v /path/to/PrairieLearn:/PrairieLearn prairielearn/prairielearn /usr/local/bin/npm ci
 ```
 
+The path `/path/to/PrairieLearn` should be replaced with the *absolute* path to the PrairieLearn source on your computer.  If you're in the root of the source directory already, you can substitute `%cd%` (on Windows) or `$PWD` (Linux and MacOS) for `/path/to/PrairieLearn`.
+
 * Create the file `PrairieLearn/config.json`:
```

Comment:
Actually, since #2158 this step is not required. PL will use a default `courseDirs` with just the `exampleCourse` if `config.json` is missing. Maybe we should delete this step from the main instruction sequence but add a note saying that PL will use the exampleCourse by default?

Actually, maybe we should change the default (in `lib/config.js`) to match `docker/config.json`? That is, include `testCourse` and the `config[N]` directories? That would allow us to delete `docker/config.json` and would also make the experience using containerized version for dev more similar to the normal docker-based PL experience.

While we are there, I'm wondering whether we should also change the default for `filesRoot` to match `docker/config.json`. Are there downsides with this?

Reply:
Absolutely, I think removing the config.json step is a great idea.  I've briefly looked through the source and I don't see any immediate downsides to giving a value for `filesRoot`, so I'll go about updating the values in `lib/config.js`.